src/SeoInspector.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+
+import fs from 'fs';
+import { JSDOM } from 'jsdom';
+import _ from 'lodash';
+import conv from 'iconv-lite';
+import RuleFactory from './RuleFactory';
+
+const defaultRules = {
+    'ImgTagWithAltAttritube': { enabled: 1 },
+    'ATagWithRelAttritube': { enabled: 1 },
+    'HeadTagWithTitleAndDescriptionsKeywordsMeta': { enabled: 1 },
+    'NoTooManyStrongTags': { enabled: 1, threshold: 15 },
+    'NoMoreThanOneH1Tag': { enabled: 1 }
+};
+
+class SeoInspector {
+
+    constructor(options = {}) {
+        this.done = (options.done) ? options.done : (err) => { if (err) throw err };
+        this.input = null;
+        this.output = null;
+        this.rules = {};
+        this.ruleFactory = new RuleFactory();
+        return this;
```

Comment:
Why return 'this' ref ?

Reply:
for chaining
new Inspector().read().addRule().write();